### DISCUSSION TITLE
请教skynet里用到bson.timestamp之类型数据输出日志时导致乱码怎么处理？

### DISCUSSION THREAD

**Discussion by 253980289** [cloudwu/skynet#1986] [Q&A]
请教skynet里用到bson.timestamp之类型数据输出日志时导致乱码怎么处理？如何每个数据都要做特别处理会比较麻烦，不好写通用接口。

**Comment by cloudwu** [comment]
用 bson.type() 逐个元素转换。

**Reply by 253980289** [reply]
感谢，这个知道，只是在通用公共接口中，不知道具体的数据结构，比如通用的从数据库返回结果打印日志时。
不过这个问题只在开发阶段的终端控制台中显示日志才会显示乱码，上线后服务是后台运行并看日志文件的，不会影响到正常看日志。
